src/Main/Main.make

```diff
@@ -210,7 +210,9 @@ else
 	sed -e 's/_VERSION_/$(patsubst %a,%.1,$(patsubst %b,%.2,$(TC_VERSION)))/' ../Build/Resources/MacOSX/Info.plist.xml >$(APPNAME).app/Contents/Info.plist
 endif
 	chmod -R go-w $(APPNAME).app
+ifneq ($(LOCAL_DEVELOPMENT_BUILD),"true")
```

Comment:
```suggestion
ifneq ("$(LOCAL_DEVELOPMENT_BUILD)","true")
```
This needs to be quoted to work. Also I might've chosen something shorter like NOSIGN which makes it also clear what the flag does, just skips the package sign parts.

Reply:
If we do not sign, the installer name in the later part will have wrong name that does not exist:

https://github.com/veracrypt/VeraCrypt/blob/6e28375060e043e9039bac4d292ecbcc5e94b08d/src/Main/Main.make#L233C2-L233C2

So, for no signing path, the copied installer name should be `"$(BASE_DIR)/Setup/MacOSX/VeraCrypt $(TC_VERSION).pkg"` instead of `"$(BASE_DIR)/Setup/MacOSX/VeraCrypt_$(TC_VERSION).pkg"`

Reply:
Thanks! I'll have a look at this (and possibly others) once the maintainer reviews